airflow/contrib/operators/postgres_to_gcs_operator.py

```diff
@@ -134,6 +134,7 @@ def _write_local_data_files(self, cursor):
         """
         schema = list(map(lambda schema_tuple: schema_tuple[0], cursor.description))
         file_no = 0
+        row_no = 1
```

Comment:
maybe a dump question, but why we initialize the variable as 1 not 0? I am not very familiiar with this operator but if the  cursor doesn't return any row, the log will still print `Received 1 rows`?

Reply:
Typo I guess, this should be 0 indeed, thanks!